tests/test-translations.php

```diff
@@ -133,4 +133,18 @@ public function test_term_translations() {
 
 	}
 
+	public function test_canonical_content_fallback() {
+		$this->assertSame( 'en_US', get_locale() );
+
+		$en = $this->factory->post->create_and_get();
+		// In normal operation, the create_empty_translations method is called on an immediate single cron job
+		$GLOBALS['bbl_jobs']->create_empty_translations($en->ID);
```

Comment:
This will need changing to the following when #312 is merged:

`Babble::get( 'jobs' )->create_empty_translations( $en->ID );`